rfcs/disable_notify_test_state.md

```diff
@@ -0,0 +1,20 @@
+# RFC ??: Add a new test property to not displaying test state
+
+## Summary
+
+Add a new test property called `notify_test_state` to not displaying
+test state when tests are running.
+
+## Details
```

Comment:
How does this interact with the `test_done_callbacks`? I believe those too cause `WindowTestEnvironment` to update the DOM, right?

Reply:
I don't think this impacts the `test_done_callbacks`, this PR just prevents `notify_test_state` to be invoked. We invoke the `test_done_callbacks` in `notify_result` function which is not impacted by this PR.

Reply:
Yes, what I'm asking is if they could affect the tests! E.g. if a test has 10 subtests, aren't the `test_done_callbacks` invoked for each individual subtest finishing, and then each of those 10 updates causes DOM changes?

Put another way, I suspect that blocking just `notify_test_state` may not be sufficient.